datapack/src/main/resources/movecraft-data/data/minecraft/tags/entity_types/hostiles.json

```diff
@@ -0,0 +1,3 @@
+{
+  "values": ["elder_guardian", "wither_skeleton", "stray", "husk", "zombie_villager", "evoker", "vex", "vindicator", "illusioner", "creeper", "skeleton", "spider", "giant", "zombie", "zombified_piglin", "enderman", "cave_spider", "silverfish", "blaze", "wither", "witch", "endermite", "guardian", "drowned", "pillager", "ravager", "piglin", "zoglin", "piglin_brute", "warden", "breeze", "bogged"]
```

Comment:
I feel like this should be provided for us.

Reply:
There is no other way to get the hostile mobs list, the other way would be class comparisons which have been ruled out, this is because mojang abandoned the concept of "hostile mobs" long ago